pyro/distributions/multinomial.py

```diff
@@ -25,6 +25,11 @@ class Multinomial(Distribution):
     def __init__(self, ps, n, batch_size=None, *args, **kwargs):
         if ps.dim() not in (1, 2):
             raise ValueError("Parameter `ps` must be either 1 or 2 dimensional.")
+        if isinstance(n, int):
```

Comment:
Prefer `isinstance(n, (int, long))` or `isinstance(n, numbers.Number)`.

Reply:
Thanks! I prefer your second suggestion. In python 3, there is no `long` type.